### DISCUSSION TITLE
Is there any plans for implementing a native search a node?

### DISCUSSION THREAD

**Discussion by Anderlyn** [xyflow/xyflow#4708] [Ideas]
Today when using CTRL + F to search in a large tree, it finds the match but it doesn't support zooming into the node that has been searched. 

Are there any plans to support a search component (overriding the browser functionality, like Figma)? I would be happy to contribute a component for this, but I wanted to know if this is something that has been considered.

**Comment by bcakmakoglu** [comment]
I don't see why this would have to be part of the core library.
Stuff like this is solving very specific use-cases which the XYFlow library is not meant to do, it's meant to provide you the *tools* to solve these use-cases yourself.

An example for the docs would be cool though ^^

**Reply by Anderlyn** [reply]
Sounds good! I will add an example If I end up working on one.
